### PR TITLE
(cargo-release) version 0.4.1

### PR DESCRIPTION
**Pull request by rfuest**
While I was able to push a tag to the `tinybmp` master branch, I'm not allowed to push commits because of this:
grafik

### CODE REVIEW

**Review by jamwaffles — APPROVED** [review]
Argh that's frustrating. I think the only way to let you do that is to make you an administrator, or we continue to go with the pull request method as we've done so far. Thoughts?

**Comment by rfuest** [comment]
The pull request method is too error prone IMO, because it doesn't support tags. The commit SHA changes when the PR is merged I need to manually recreate the release tag after the PR is merged.

I think we have three options:
* You change my role to "Maintain" (I think "Admin" isn't necessary)
  I don't know if you are comfortable with giving me so many access rights to the projects.
* You continue to handle the releases.
  I would like to support you by taking over some of the maintenance work, but if that means that you need to give up too much control over the project this might be the only solution.
* We look into further automation of the release process.
  We might be able to use a manually triggered GH action to release the crate. This way it would only be necessary to have "Write" permissions to the repo to create a release. But it would be a lot of work to set this up and test it.

**Comment by jamwaffles** [comment]
Yeah agreed, the PR approach is a kludge. To your other points:

1. The e-g core team has the "maintain" role for at least `tinytga`. I can change this to "admin" (which is the highest level) for the repos you want to be able to push to and see how that works out next time we release. I have every reason to trust you so I will force myself to push past my irrational fears :D
2. I'm happy to do this also. Running `cargo release` isn't too much of a hassle and the process is pretty ironed out at this point, although I would very much like to raise the bus factor above 1.
3. I'd prefer not to automate releases. I guess there are two sides to human control/error, but I like being able to check and confirm things manually during releases.

**Pull request merged**
